ui/src/criteriaHolder.tsx

```diff
@@ -2,25 +2,26 @@ import ActionBar from "actionBar";
 import { CriteriaPlugin } from "cohort";
 import GridLayout from "layout/gridLayout";
 import { useState } from "react";
+import { useNavigate } from "react-router-dom";
 
 export type CriteriaHolderProps = {
   title: string;
   plugin: CriteriaPlugin<object>;
-  doneURL?: string;
   cohort?: boolean;
-  defaultBackURL?: string;
```

Comment:
Are `doneURL` and `defaultBackURL` configurable? Per deployment or per criteria plugin?

Reply:
The `exitAction` is configurable per deployment via configuration in router.tsx. That is potentially insufficient for some deployments with multiple underlays but I expect all non-trivial deployments to embed Tanagra in an iframe, which case the exit action will always be sending a message to close the iframe.